### DISCUSSION TITLE
Allow Chair Lift ride to be built in a loop

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22688] [Feature requests]
Chair Lift ride can now only be built from point to point.   But it would be nice to allow a loop design as with monorails or trains.   I tried a loop and it is rejected when a test is attempted.

**Comment by astraylife1** [comment]
But then the chairlift cars would only be on one side of the track. Unless there is a new piece that allows for a turnaround without a station. 

You can somewhat get around this by building a ride with two stations but the second station only has an entrance. Use tile inspector to delete the entrance. Since the only breakdown for chairlift is safety cut out, as long as the station with no entrance is not station 1, the mechanics have no need to access the station and guests don't load there either.
